hw_29_drinks/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HW29 DRINKS LIST</title>
+    <script src="./js/letElemli.js" defer></script>
+</head>
+
+<body>
+    <div><h2 id="drinks-header"></h2></div>
+    <div><ul id="drinks-list"></ul></div>
+</body>
```

Comment:
Wrong understanding of task. Condition was to get `<li>` elements of the list. 
So your html should look like
```html
<h2>Drinks</h2>
<ul id="drinks-list">
  <li>Coca-Cola</li>
  <li>Tea</li>
  <li>Soda</li>
  <li>Milk</li>
  <li>Mineral Water</li>
  <li>Tea</li>
  <li>Coffee</li>
  <li>Milk</li>
  <li>Pepsi</li>
  <li>Coca-Cola</li>
  <li>Coffee</li>
</ul>
```